packages/applicant/src/middleware.page.ts

```diff
@@ -45,6 +45,16 @@ export function buildMiddlewareResponse(req: NextRequest, redirectUri: string) {
         maxAge: 900,
       }
```

Comment:
do we need to add the api/ path to line 22?

Reply:
yep probs